### PR TITLE
Fix the incompatible replay dialogue translations

### PR DESCRIPTION
**Pull request by abcdefg30**
Updates Linguini to include the changes from https://github.com/Ygg01/Linguini/pull/24, and adds some new lines to stop the text from overflowing over the dialogue borders:
grafik
grafik
grafik
grafik

### CODE REVIEW

File: mods/common/languages/en.ftl [hunk 1]
```diff
@@ -502,11 +502,16 @@ replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug
...
-incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod: { $mod }.
-incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } an unavailable mod: { $mod }.
-incompatible-replay-incompatible-version = { -incompatible-replay-recorded } an incompatible version: { $version }.
-incompatible-replay-unavailable-map = { -incompatible-replay-recorded } an unavailable map: { $map }.
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version:
+    { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod:
+    { $mod }.
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } an unavailable mod:
+    { $mod }.
```
Comment (PunkPun) [inline mods/common/languages/en.ftl]: Perhaps keep `$mod` in the same line? We use short names and it looks out of place to have them in a newline

**Review by RoosterDragon — DISMISSED** [review]
Thanks for these changes, nice tidyup.

File: mods/common/languages/en.ftl [hunk 2]
```diff
@@ -502,11 +502,16 @@ replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug
...
 -incompatible-replay-recorded = It was recorded with
-incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version: { $version }.
-incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod: { $mod }.
-incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } an unavailable mod: { $mod }.
-incompatible-replay-incompatible-version = { -incompatible-replay-recorded } an incompatible version: { $version }.
-incompatible-replay-unavailable-map = { -incompatible-replay-recorded } an unavailable map: { $map }.
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version:
+    { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod:
+    { $mod }.
```
Comment (dragunoff) [inline mods/common/languages/en.ftl]: In these cases `$version` and `$mod` are `null` so the game crashes:
```
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object.
   at OpenRA.TranslationExts.ToFluentType(Object value) in /home/ivaylo/code/OpenRA/OpenRA.Game/TranslationExts.cs:line 41
   at OpenRA.Translation.TryGetString(String key, String& value, IDictionary`2 arguments) in /home/ivaylo/code/OpenRA/OpenRA.Game/Translation.cs:line 100
   at OpenRA.Mods.Common.Widgets.ConfirmationDialogs.ButtonPrompt(ModData modData, String title, String text, Dictionary`2 titleArguments, Dictionary`2 textArguments, Action onConfirm, Action onCancel, Action onOther, String confirmText, String cancelText, String otherText) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs:line 43
   at OpenRA.Mods.Common.Widgets.Logic.ReplayUtils.IncompatibleReplayDialog(String text, Dictionary`2 textArguments, ModData modData, Action onCancel) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayUtils.cs:line 78
   at OpenRA.Mods.Common.Widgets.Logic.ReplayUtils.PromptConfirmReplayCompatibility(ReplayMetadata replayMeta, ModData modData, Action onCancel) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayUtils.cs:line 62
   at OpenRA.Mods.Common.Widgets.Logic.ReplayBrowserLogic.WatchReplay() in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayBrowserLogic.cs:line 771
```

We don't need to pass any arguments - the mod/version is simply unknown.

File: mods/common/languages/en.ftl [hunk 3]
```diff
@@ -502,11 +502,16 @@ replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug
...
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } an unavailable mod:
+    { $mod }.
+incompatible-replay-incompatible-version = { -incompatible-replay-recorded } an incompatible version:
+    { $version }.
+incompatible-replay-unavailable-map = { -incompatible-replay-recorded } an unavailable map:
+    { $map }.
```
Comment (dragunoff) [inline mods/common/languages/en.ftl:512]: I wasn't able to trigger this one, the game tries to download the map from the resource center and if it fails it won't even allow the "Watch" button to be clicked.

**Review by dragunoff — CHANGES_REQUESTED** [review]
I think it makes more sense for the button to say "OK" instead of "Cancel":
image

Sorry for the scope creeping but I noticed some issues around this.

File: mods/common/languages/en.ftl (hunk 3 above)
Comment (dragunoff) [reply mods/common/languages/en.ftl:512]: Ah, I guess this is for when we load directly into a replay.

**Comment by abcdefg30** [comment]
Updated:
- Changed "Cancel" to "OK"
grafik
- Removed line breaks for mod names
grafik
- Fixed the crash with null arguments
grafik

**Review by dragunoff — APPROVED** [review]

**Review by PunkPun — APPROVED** [review]

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/6cd6fa16ef828d492415130e5bafe0ae61d1957a...607e3b63393dec981f1ba65c6675029adb37ab0d)

**Pull request merged**